cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
+| <<administration-databases-drop-database, DROP DATABASE ... [IF EXISTS]>>                                                                                                         | Database administration       | Deletes a specified database.
+| <<administration-databases-start-database, START DATABASE>>                                                                                                                       | Database administration       | Starts up a specified database.
+| <<administration-databases-stop-database, STOP DATABASE>>                                                                                                                         | Database administration       | Stops a specified database.
+| <<administration-security-roles-show, SHOW [ALL \| POPULATED] ROLES [WITH USERS]>>                                                                                                | User and role administration  | Returns information about all or populated roles, optionally including the assigned users.
+| <<administration-security-roles-create, CREATE [OR REPLACE] ROLE ... [IF NOT EXISTS] ... [AS COPY OF]>>                                                                           | User and role administration  | Creates new roles.
+| <<administration-security-roles-drop, DROP ROLE [IF EXISTS]>>                                                                                                                     | User and role administration  | Deletes a specified role.
+| <<administration-security-users-show, SHOW USERS>>                                                                                                                                | User and role administration  | Returns information about all users.
+| <<administration-security-users-create, CREATE [OR REPLACE] USER ... [IF NOT EXISTS] SET PASSWORD ... [SET PASSWORD CHANGE [NOT] REQUIRED] [SET STATUS [ACTIVE \| SUSPENDED]]>>   | User and role administration  | Creates a new user and sets the password for the new account. Optionally the account status can also be set and if the user should change the password upon first login.
+| <<administration-security-users-drop, DROP USER ... [IF EXISTS]>>                                                                                                                 | User and role administration  | Deletes a specified user.
+| <<administration-security-users-alter, ALTER USER ... [SET PASSWORD ...] [SET PASSWORD CHANGE [NOT] REQUIRED] [SET STATUS [ACTIVE \| SUSPENDED]]>>                                | User and role administration  | Changes a user account. Changes can include setting a new password, setting the account status and enabling that the user should change the password upon next login.
```

Comment:
Same for ACTIVE/SUSPENDED here as for CREATE USER.

When wanting to change both the password and if it needs to be changed again (since that does not happen automatically) you can skip the second `SET PASSWORD`. I don't know how we best captures it but something like `ALTER USER ... [SET PASSWORD {password [CHANGE [NOT] REQUIRED] | CHANGE [NOT] REQUIRED}] [SET STATUS {ACTIVE | SUSPENDED}]` maybe? 

I think we in most places just have `[SET PASSWORD ... [CHANGE [NOT] REQUIRED]] [SET PASSWORD CHANGE [NOT] REQUIRED]` but that would be slightly wrong since you can't have `SET PASSWORD password CHANGE REQUIRED SET PASSWORD CHANGE NOT REQUIRED`

Reply:
The documentation section about this have:

```
ALTER USER name SET {
      PASSWORD password
            [[SET PASSWORD] CHANGE [NOT] REQUIRED]
            [SET STATUS {ACTIVE | SUSPENDED} ] |
      PASSWORD CHANGE [NOT] REQUIRED
            [SET STATUS {ACTIVE | SUSPENDED}] |
      STATUS {ACTIVE | SUSPENDED}
}
```

Reply:
I am going with your first option